frontend/public/components/utils/details-page.tsx

```diff
@@ -37,7 +37,9 @@ const getTolerationsPath = (obj: K8sResourceKind): string => {
 
 const getNodeSelectorPath = (obj: K8sResourceKind): string => {
   // FIXME: Is this correct for all types (jobs, cron jobs)? It would be better for the embedding page to pass in the path.
-  return obj.kind === 'Pod' ? 'spec.nodeSelector' : 'spec.template.spec.nodeSelector';
+  return obj.kind === 'Pod' || obj.kind === 'MachineConfigPool'
+    ? 'spec.nodeSelector'
+    : 'spec.template.spec.nodeSelector';
```

Comment:
Hm, I forgot about this.

It would be better to pass the path in like we do for `podSelector` instead of hard-coding types here.

Reply:
Hi @spadgett, fixed.